test/integration/stats_integration_test.cc

```diff
@@ -287,7 +288,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // If you encounter a failure here, please see
   // https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md#stats-memory-tests
   // for details on how to fix.
-  EXPECT_MEMORY_EQ(m_per_cluster, 44491);
+  EXPECT_MEMORY_EQ(m_per_cluster, 44747);
```

Comment:
This 256 byte difference seems surprising. I would think that it would just be a 64-byte overhead for the unique_ptr you added if this setting was not explicitly configured. Are there 4 of these structures by default?

I'm wondering if the structure is being instantiated by default despite the code defending against doing so, so we'd have the 64 bytes for the pointer, plus:
```
  const std::string runtime_key_;    // 24 bytes plus more if runtime_key > 22 chars
  const double default_value_;       // 8 bytes
  Runtime::Loader& runtime_;         // 8 bytes
```
This looks like 104 bytes plus (if the runtime key is large, another 64 bytes and the size of the runtime key).

What's the runtime key? That wasn't obvious to me on inspection.

Reply:
There is no default value for the runtime key. It can be defined via CDS so that the bias can be overridden via runtime.

Users can specify a single runtime key for all clusters if they want to be able to easily change the bias of all clusters, or they can specify a runtime key that contains the cluster name so that they can set per-cluster biases.